decontamination/README.md

```diff
@@ -22,3 +22,23 @@ The results from a previous decontamination run can be used to speed-up the scri
 ```bash
 python find_substrings.py --dataset-name bigcode/stack-dedup-alt-filter-no-pii --output-dir /path/to/output  --num-proc 32 --cached-decontamination-dir /path/to/previous/output/ --cache-retrieval-key content --split-languages
 ```
+
+## Near Matching with MinHash and LSH
+
+This is similar to the near deduplication script `data_analysis/near-deduplication/minhash_deduplication_alt.py` with one modification: we use benchmark datasets as index source instead of the dataset itself.
```

Comment:
```suggestion
Instead of looking for exact matches from the test sets, we look for near duplicates. This is similar to the near deduplication script [`data_analysis/near-deduplication/minhash_deduplication_alt.py`](https://github.com/bigcode-project/bigcode-analysis/blob/main/data_analysis/near-deduplication/minhash_deduplication_alt.py) with one modification: we use benchmark datasets as index source instead of the dataset itself.
```